### PR TITLE
Update create-users.md

### PR DESCRIPTION
**Pull request by AlexSlawinski**
Minor change in prep for https://spryker.atlassian.net/browse/CC-26261 guide

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 1]
```diff
@@ -44,8 +44,9 @@ This document describes how to create users in the Back Office.
 7. Enter a **LAST NAME**.
 8. For **ASSIGNED GROUPS**, select one or more user groups you want to assign this user to.
 9. Optional: To make this user an agent, select **THIS USER IS AN AGENT**.
-10. Select an **INTERFACE LANGUAGE**.
-11. Click **Create**.
+10. Optional: to make this user a warehouse user, select **IS A WAREHOUSE USER**.
```
Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:47]:
```suggestion
10. Optional: Ensure the user has the **WAREHOUSE USER** role.
```

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 2]
```diff
@@ -44,8 +44,9 @@ This document describes how to create users in the Back Office.
...
+11. Select an **INTERFACE LANGUAGE**.
```
Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:48]:
```suggestion
10. Optional: to make this user a warehouse user, select **IS A WAREHOUSE USER**.
11. Select an interface language.
```

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 3]
```diff
@@ -44,8 +44,9 @@ This document describes how to create users in the Back Office.
 7. Enter a **LAST NAME**.
 8. For **ASSIGNED GROUPS**, select one or more user groups you want to assign this user to.
 9. Optional: To make this user an agent, select **THIS USER IS AN AGENT**.
-10. Select an **INTERFACE LANGUAGE**.
-11. Click **Create**.
+10. Optional: Ensure the user has the **WAREHOUSE USER** role.
+11. Select an **INTERFACE LANGUAGE**.
```
Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:48]:
```suggestion
11. Select an interface language.
```

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 4]
```diff
@@ -44,8 +44,9 @@ This document describes how to create users in the Back Office.
 7. Enter a **LAST NAME**.
 8. For **ASSIGNED GROUPS**, select one or more user groups you want to assign this user to.
 9. Optional: To make this user an agent, select **THIS USER IS AN AGENT**.
-10. Select an **INTERFACE LANGUAGE**.
-11. Click **Create**.
+10. Optional: Ensure the user has the **WAREHOUSE USER** role.
+11. For **INTERFACE LANGUAGE**, select a language suitable for the user.
+12. **Create** a user.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:49]:
```suggestion
12. Click **Create**.
```

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 5]
```diff
@@ -44,8 +44,9 @@ This document describes how to create users in the Back Office.
 7. Enter a **LAST NAME**.
 8. For **ASSIGNED GROUPS**, select one or more user groups you want to assign this user to.
 9. Optional: To make this user an agent, select **THIS USER IS AN AGENT**.
-10. Select an **INTERFACE LANGUAGE**.
-11. Click **Create**.
+10. Optional: Ensure the user has the **WAREHOUSE USER** role.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:47]: Why? How?

Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:47]:
```suggestion
10. Optional: Ensure the user has the warehouse user role by clicking **WAREHOUSE USER**.
```

File: docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md [hunk 6]
```diff
@@ -61,3 +62,4 @@ This opens the **Users** page with the success message displayed. The created us
 | ASSIGNED GROUPS | User groups to assign this user to. User groups define what areas and actions the user will have access to. To learn how to create user groups, see [Create user groups](/docs/pbc/all/user-management/{{page.version}}/manage-in-the-back-office/manage-user-groups/create-user-groups.html). |
 | AGENT | Defines if this user is an [agent assist](/docs/pbc/all/user-management/{{page.version}}/agent-assist-feature-overview.html) |
 | INTERFACE LANGUAGE | Defines the interface language of the Back Office for this user. |
+| WAREHOUSE USER| Defines if this user is a warehouse user. |
```
Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202212.0/manage-in-the-back-office/manage-users/create-users.md:65]:
```suggestion
| WAREHOUSE USER| Defines if this user is a warehouse user. A warehouse user is someone who works inside a warehouse to fulfill orders. |
```

**Pull request merged**